macOS/DuckDuckGo/Subscription/SubscriptionUIHandler.swift

```diff
@@ -58,10 +59,10 @@ final class SubscriptionUIHandler: SubscriptionUIHandling {
         let actionHandlers = SubscriptionAccessActionHandlers(openActivateViaEmailURL: {
             let url = Application.appDelegate.subscriptionAuthV1toV2Bridge.url(for: .activationFlow)
             handler.subscriptionAccessActionOpenURLHandler(url: url)
+            PixelKit.fire(PrivacyProPixel.privacyProRestorePurchaseEmailStart, frequency: .legacyDailyAndCount)
         }, restorePurchases: {
             handler.subscriptionAccessActionRestorePurchases(message: message)
-        }, uiActionHandler: { event in
```

Comment:
I was able to get rid of passing whole `uiActionHandler` and explicitly fire appropriate pixels since there are only 2 specific actions to be handled.

Reply:
Nice!